### PR TITLE
support customer tags

### PR DESCRIPTION
**Pull request by njuettner**
We want to add additional tags from customers.
https://github.com/giantswarm/roadmap/issues/984

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: pkg/irsa/irsa.go [hunk 1]
```diff
@@ -183,3 +198,14 @@ func (s *IRSAService) Delete(ctx context.Context) error {
 func toDeletePropagation(v metav1.DeletionPropagation) *metav1.DeletionPropagation {
 	return &v
 }
+
+func getCustomerTags(cluster *capi.Cluster) map[string]string {
+	customerTags := make(map[string]string)
+
+	for k, v := range cluster.Labels {
+		if strings.HasPrefix(k, key.CustomerTagLabel) {
+			customerTags[k] = v
```
Comment (calvix) [inline pkg/irsa/irsa.go:207]: the key of the tag itself should be stripped of the prefix

Comment (calvix) [reply pkg/irsa/irsa.go:207]: `tag.provider.giantswarm.io/test=test" -> `test=test`

**Comment by calvix** [comment]
is there a way how we can tag this resource?
https://github.com/giantswarm/irsa-operator/blob/master/pkg/irsa/irsa.go#L133

**Review by whites11 — APPROVED** [review]

**Review by calvix — APPROVED** [review]
lgtm

**Pull request merged**